sdk/analysisservices/Azure.ResourceManager.Analysis/api/Azure.ResourceManager.Analysis.netstandard2.0.cs

```diff
@@ -0,0 +1,288 @@
+namespace Azure.ResourceManager.Analysis
```

Comment:
As we already use Analysis as the namespace instead of AnalysicServices, should we also update the renaming to keep consistent?

Reply:
fixed